source/install/mattermost-kubernetes-operator.rst

```diff
@@ -180,6 +180,141 @@ The process described below needs to be completed prior to proceeding with the M
+  $ kubectl get secrets -n mattermost mattermost-license-secret -o yaml > mattermost-license-secret.yaml
+  $ kubectl get secrets -n cert -o yaml > cert.yaml
+
+3. Change the db secret to match operator, the helm used mattermost.dbsecret while the operator uses DB_CONNECTION_STRING
+
+.. code-block:: sh
+
+  $ vim mattermost-db-secret.yaml
+
+.. code-block:: yaml
+
+  apiVersion: v1
+  kind: Secret
+  metadata:
+    name: mattermost-db-secret
+  data:
+    DB_CONNECTION_STRING: "db_connection_string"
+
+4. Create S3 Secret, mattermost operator uses secret while helm uses environment variables
+
+.. code-block:: yaml
+
+  apiVersion: v1
+  metadata:
+    name: s3-secret
+  kind: Secret
+  stringData:
+    accesskey: accesskey
+    secretkey: secretkey
+
+5. Scale the current installation to 0 and create the operator
+
+.. code-block:: sh
+
+  $ kubectl apply -n mattermost-operator -f https://raw.githubusercontent.com/mattermost/mattermost-operator/master/docs/mattermost-operator/mattermost-operator.yaml
+  $ kubectl scale deployment -n mattermost mattermost-deployment --replicas=0
+
+6. Apply the new secrets in the new namespace 
+
+.. code-block:: sh
+
+  $ kubectl apply -f mattermost-db-secret.yaml -n mattermost-operator 
+  $ kubectl apply -f mattermost mattermost-license-secret.yaml -n mattermost-operator
+  $ kubectl apply -f cert.yaml -n mattermost-operator
+  $ kubectl apply -f s3-secret -n mattermost-operator
+
+7. Create a mattermost installation according to the secrets imported and exported to the new ns
+
+.. code-block:: yaml
+
+  apiVersion: installation.mattermost.com/v1beta1
+  kind: Mattermost
+  metadata:
+    name: mm-example
+  spec:
+    image: mattermost/mattermost-enterprise-edition
+    imagePullPolicy: IfNotPresent
+    version: 6.0.0
+    size: 5000users
+    ingress:
+      enabled: true
+      host: example.mattermost-example.com
+      tlsSecret: "my-tls"
+    licenseSecret: "license-secret"
+    database:
+      external:
+        secret: db-secret
+    fileStore:
+      external:
+        url: s3.amazonaws.com
+        bucket: my-s3-bucket
+        secret: s3-secret
+    elasticSearch: {}
+    volumeMounts:
+    - name: certificates
+      mountPath: /etc/ssl/certs/mycert.pem
+      subPath: mycert.pem
+    volumes:
+    - name: certificates
+      secret:
+        defaultMode: 420
+        secretName: cert
+
+Save the file as mattermost-installation.yaml. While recommended file names are provided, your naming conventions may differ.
+
+8. Apply the new manifest in the relavent namespace 
+
+.. code-block:: sh
+
+  $ kubectl apply -n mattermost -f mattermost-installation.yaml
+
+The deployment process can be monitored in the Kubernetes user interface or in command line by running:
+
+.. code-block:: sh
+
+  $ kubectl -n mattermost get mm -w
+
+The installation should be deployed successfully, when the Custom Resource reaches the stable state.
+
+9. Configure DNS and use Mattermost.
```

Comment:
This step is the same as regular installation, perhaps we could link to it? Not sure it is possible within the `rst` file.